inc/cf7.php

```diff
@@ -49,6 +52,18 @@ function klypHsCf7CatchSubmission($result, $tags)
     $hubspot->cf7FormId     = $cf7FormId;
     $hubspot->cf7FormFields = $cf7FormFields;
     $hubspot->hsFormFields  = $hsFormFields;
+
+    if ($files) 
+    {
+        //call function hsFileUpload() that uploads all files on hubspot.
+        $hsFilesPath       = $hubspot->hsFileUpload($files);
+        //$hsFilesPath is an array that contains path of hubspot files.
+        if ($hsFilesPath)
+        {
+        // merge the uploaded file of hubspot path with 
+            $_POST         = array_merge($_POST, $hsFilesPath); 

```

Comment:
$_POST = array_merge($_POST, $hsFilesPath); 

please fix for all subsequent **_single_** variable line assignments